gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -169,18 +169,12 @@ def main(args):
     # Prefix to create custom named versions of outputs
     prefix = args.prefix + version
 
-    input_paths_dict = {
-        "v4.0_exomes": v4_input_ht().path,
-        "test_v4.0_exomes": v4_input_ht().path,
-    }
+    input_path = v4_input_ht().path
 
-    output_paths_dict = {
-        "v4.0_exomes": v4_vrs_annotations(annotated=True).path,
-        "test_v4.0_exomes": v4_vrs_annotations(test=True, annotated=True).path,
-    }
+    output_path = v4_vrs_annotations(test=args.test, original_annotations=True)
```

Comment:
this can be moved down to right after "if args.annotate_original:" because it's only needed  if running annotate_original